### PR TITLE
PvMTickCounter

### PR DESCRIPTION
**Pull request by Chrisioman**
Adds a variety of tools to calculate individual damage dealt based of seconds or combat ticks.
Can keep track of how many max hits are landed
Also counts individual combat ticks versus others in the same instance based off weapon used.

### CODE REVIEW

**Comment by LlemonDuck** [comment]
You need to place your plugin file within the plugins folder, rather than in the root of the repository. Also, use lower-kebab-case for the file name.

**Comment by Chrisioman** [comment]
my bad, should have doubled checked that. I think I made the appropriate changes

**Comment by Nightfirecat** [comment]
Bot seems confused, closing and reopening to see if it'll wake up

File: plugins/pvmtickcounter [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ repository=https://github.com/Chrisioman/PvMTickCounter.git
+ commit=85ad93e11fe34a1857e21bba4bc8176f2b638e82
```
Comment (Nightfirecat) [inline plugins/pvmtickcounter:2]: You have a leading space before each of these lines, remove that.
```suggestion
repository=https://github.com/Chrisioman/PvMTickCounter.git
commit=85ad93e11fe34a1857e21bba4bc8176f2b638e82
```

**Pull request merged**